### DISCUSSION TITLE
AdmissionWebhooks Is not called anymore on stream close

### DISCUSSION THREAD

**Discussion by Pernifloss** [OvenMediaLabs/OvenMediaEngine#1371] [Help]
Hello, I upgraded from 0.15.11 to 0.15.15 last week, and the `AdmissionWebhooks` is not called anymore when the stream close.

I was using OME 0.15.11 and it was working fine.

Do I need to update my configuration, or is that a bug, or the new version doesn't call webhook anymore on stream closing?

```
            <AdmissionWebhooks>
                <ControlServerUrl>https://myserver.tv/webtv.php?command=AllowWebTVStream</ControlServerUrl>
                <Timeout>3000</Timeout>
                <Enables>
                    <Providers>rtmp</Providers>
                </Enables>
            </AdmissionWebhooks>
```

Thanks in advance.

**Comment by choigilhoon** [comment]
@Pernifloss 

Events should occur normally even when the stream is closing.

I've tested it using OBS, but I couldn't reproduce the issue.

To perform a precise analysis, please provide the following information:
1. What encoder are you using?
2. The full logs at the moment when the problem occurred.
3. ```Server.xml``` file.

**Comment by Pernifloss** [comment]
Hello, thanks for your fast reply,

It's indeed impossible to reproduce with OBS.

We use are own streaming program.

It seems to happen when the stream is closed before the first Webhook is resolved. Or when the program close or crash before the stream is closed. 

It's a weird use case, I'll find a workaround

Thanks.
